packs/nagios/etc/st2service_handler.py

```diff
@@ -130,6 +150,17 @@ def main(args):
 
 
 if __name__ == '__main__':
+    if not os.path.exists(ST2_CONFIG_FILE):
+        print('Configuration file not found. Exiting.')
```

Comment:
Already added in the docs @Kami. You +1ed it. https://github.com/StackStorm/st2/pull/1412